Error in Update Default Keybindings workflow

An error has been reported on the latest attempt of Update Default Keybindings workflow.

https://github.com/tshino/vscode-kb-macro/actions/runs/2980662375
```
Run npm run gen-wrapper

> kb-macro@0.12.[5](https://github.com/tshino/vscode-kb-macro/runs/8160826728?check_suite_focus=true#step:5:6) gen-wrapper
> node ./generator/gen_wrapper.js


> kb-macro@0.12.5 postgen-wrapper
> npm run verify-wrapper


> kb-macro@0.12.5 verify-wrapper
> node ./generator/verify_wrapper.js

TypeError: Cannot read properties of undefined (reading 'length')
    at joinComplementalKeybindings (/home/runner/work/vscode-kb-macro/vscode-kb-macro/generator/verify_wrapper.js:1[6](https://github.com/tshino/vscode-kb-macro/runs/8160826728?check_suite_focus=true#step:5:7)0:51)
    at verifyWrapper (/home/runner/work/vscode-kb-macro/vscode-kb-macro/generator/verify_wrapper.js:206:5)
Error: Process completed with exit code 1.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 All notable changes to the Keyboard Macro Bata extension will be documented in this file.
 
+## [Unreleased]
+- Internal:
+  - Fixed: Failure in automated workflow to update default keybindings wrappers. [#145](https://github.com/tshino/vscode-kb-macro/issues/145)
+
 ### [0.12.5] - 2022-08-28
 - Fix
   - Fixed: `ctrl+shift+f` fails to focus Search in recording mode. [#142](https://github.com/tshino/vscode-kb-macro/issues/142)

--- a/generator/verify_wrapper.js
+++ b/generator/verify_wrapper.js
@@ -125,6 +125,7 @@ const joinComplementalKeybindings = function(keybindings) {
             k1.mac === k2.mac &&
             k1.command === k2.command &&
             k1.args && k2.args &&
+            'await' in k1.args && 'await' in k2.args &&
             k1.args.command === k2.args.command) {
             const w1 = k1.when;
             const w2 = k2.when;

